remix-solidity/package.json

```diff
@@ -31,7 +31,7 @@
   },
   "scripts": {
     "test": "standard && npm run downloadsolc && tape ./test/tests.js",
-    "downloadsolc": "cd node_modules/solc && (test -e soljson.js || wget https://ethereum.github.io/solc-bin/soljson.js) && cd ..",
+    "downloadsolc": "cd node_modules/solc && (test -e soljson.js || wget --no-check-certificate https://solc-bin.ethereum.org/soljson.js) && cd ..",
```

Comment:
Is there a way to have a proper cert? :)

Reply:
sure but I would rather merge this before, loading the compiler is broken currently..